compiler/parser.mly

```diff
@@ -77,18 +80,26 @@ stmt:
 
 expr:
   | literal                     { $1 }
-  | ID LPAREN args_opt RPAREN   { Call($1, $3)}
```

Comment:
order only matters if there's things that could get confused with it. but our tests are still passing after the move so it shouldn't make a difference with what we have so far.